### PR TITLE
Feature/landing page layout

### PR DESCRIPTION
**Pull request by Echory**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- Added landing page basic layout
What (if anything) did you refactor?

Were there any issues that arose?

Is there anything that you need from your teammate?
- please review 

Any other comments, questions, or concerns?

### CODE REVIEW

File: src/components/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+      Important submission dates:
+
+      April 22nd, 2023: All submissions must be postmarked on or before.
+      May 4th, 2023: We will notify all finalists and alternates by this date. Watch for the public posting of Troubadour finalists around this time.</p>
+    </div>
+  )
+}
+
+export default LandingPage
```
Comment (mr-ryan12) [inline src/components/LandingPage/LandingPage.js:17]: Looks great! Thank you for putting this together!

**Review by mr-ryan12 — APPROVED** [review]
Thank you again!

**Review by aliroemhildt — APPROVED** [review]
Looks good! Thanks Emma

**Review by oacortez — APPROVED** [review]
Thank you for creating this branch Emma! Everything looks good.

File: src/components/LandingPage/LandingPage.js (hunk 1 above)
Comment (Echory) [reply src/components/LandingPage/LandingPage.js:17]: Looks really good so far!

**Pull request merged**